server/sandbox/sandbox.go

```diff
@@ -1,4 +1,4 @@
-package server
+package sandbox
```

Comment:
We still need Sandboxes per the discussion last week to support Clear Containers well. 
libkpod, preferably, and a sub-package, if needed, to break cycles is fine with me.